addons/event/views/event_type_views.xml

```diff
@@ -53,7 +53,7 @@
                                </tree>
```

Comment:
Commit message:

Would add a quick reference to the file and method where we use that (e.g: `See 'searchInXmlDocNotebookTab' in 'abstract_macro.js' for more details`).

Same thing for the enterprise commit.